js/scripts/test.js

```diff
@@ -0,0 +1,12 @@
+const cp = require('child_process');
+
+function execSync(commandline) {
+  cp.execSync(commandline, { stdio: 'inherit' });
+}
+
+if (process.platform !== 'win32' && process.platform !== 'darwin') {
+  execSync('yarn rebuild-slow');
+  execSync('mocha --expose-gc js/spec/index-slow-spec.js');
+  execSync('yarn rebuild');
+}
+execSync('mocha --expose-gc js/spec/index-spec.js');
```

Comment:
I had to add this script to rebuild and run tests when on Linux, couldn't figure out how to do it from package.json